docs/en-us/dev/user_doc/guide/datasource/hive.md

```diff
@@ -2,9 +2,7 @@
 
 ## Use HiveServer2
 
- <p align="center">
-    <img src="/img/hive-en.png" width="80%" />
-  </p>
+![hive](/img/new_ui/dev/datasource/hive.png)
```

Comment:
Good point to add new directory to keep new ui

Reply:
also good to use markdown syntax

Reply:
> Good point to add new directory to keep new ui

It's good for the organization and standardization of the images later on, so I chose to create a new catalog.